app/pings.yaml

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+session_end:
+  description: >
+    These pings are sent at the end of a session. (when Firefox Reality switches to the background.)
+    We usually send search and UI control metrics at the end of a session.
+  include_client_id: false
```

Comment:
Are you sure you don't want the `client_id` here? I think it would make your ping useless if you don't have it :)

Reply:
The `client_id` is the one I am not very sure. I was thinking I might not need it if I only wanna accumulate counters daily instead of recognizing these pings by `client_id` for per user. However,  including client id absolutely gives us more flexible.

Reply:
I'm fairly confident you need the `client_id` here, as you had that in your previous pings.